src/App.js

```diff
@@ -0,0 +1,111 @@
+import React from 'react';
+import './App.css';
+import resumeJson from './assets/resume.json';
+
+import Experience from "./components/Experience/Experience";
+import Study from "./components/Study/Study";
+import Award from "./components/Award/Award";
+import Skill from "./components/Skill/Skill";
+import Language from "./components/Language/Language";
+import Interest from "./components/Interest/Interest";
+import Reference from "./components/Reference/Reference";
+
+function App() {
+  const resumeData = resumeJson;
+  return (
+    <main className="App">
+      <header>
+        <div>
+          <h1>{resumeData.basics.name}</h1>
+          <span className="label">{resumeData.basics.label}</span>
+        </div>
+        <img src={resumeData.basics.picture} />
+      </header>
+      <section className="contact-info">
+        <h2>Contact Info</h2>
+        <span className="email"><i class="far fa-envelope"></i> {resumeData.basics.email}</span>
+        <span className="phone"><i class="fas fa-phone"></i> {resumeData.basics.phone}</span>
+        <span className="website">
+          <i class="fas fa-home"></i> <a href={resumeData.basics.website}>{resumeData.basics.website}</a>
+        </span>
+        <div className="location">
+          <h3><i class="fas fa-map-marker"></i> Location</h3>
+          <span className="address">{resumeData.basics.location.address}</span>
+          <span className="postal-code">{resumeData.basics.location.postalCode}</span>
+          <span className="city">{resumeData.basics.location.city}</span>
+          <span className="country-code">{resumeData.basics.location.countryCode}</span>
+          <span className="region">{resumeData.basics.location.region}</span>
+        </div>
+        <div className="profiles">
+          <h3><i class="far fa-user"></i> Profiles</h3>
+          {resumeData.basics.profiles.length ?
+            resumeData.basics.profiles.map((profile, index) => {
+              return <a key={index} href={profile.url}>{profile.network} @{profile.username}</a>
+            })
+            : ""
+          }
+        </div>
+      </section>
+      <section className="work">
+        <h2><i class="fas fa-briefcase"></i> Work</h2>
+        {resumeData.work.map((experience, index) => {
+          return <Experience key={index} type="work" experience={experience} />
+        })}
+      </section>
+      <section className="volunteer">
+        <h2><i class="fas fa-hands-helping"></i> Volunteer</h2>
+        {resumeData.volunteer.map((experience, index) => {
+          return <Experience key={index} type="volunteer" experience={experience} />
+        })}
+      </section>
+      <section className="education">
```

Comment:
I think my only suggestion is similar to what davidloesch1 said. Since the styling for all your sections is very similar, you could create a reusable 'template' component that all the sections use and pass the data into it.